javascript/builtins/Atomics.json

```diff
@@ -0,0 +1,708 @@
+{
+  "javascript": {
+    "builtins": {
+      "Atomics": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Atomics",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "60"
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
```

Comment:
16 according to the page

Reply:
The page got updated in the meantime! :speak_no_evil: I will include this.